bundle_pl.properties

```diff
@@ -0,0 +1,438 @@
+account_registration=Rejestracja konta
+account_is_verified_description=Twoje konto zostało zweryfikowane! Oznacza to, że Twoje postacie są powiązane z Twoim adresem e-mail, a jeśli zapomnisz hasła, możesz poprosić o przesłanie nowego w menu głównym.
+account_is_unverified_description=Twoje konto nie jest zweryfikowane! Oznacza to, że Twoje postacie na koncie nie są w pełni powiązane z Twoim adresem e-mail i grozi Ci automatyczne usunięcie konta, jeśli wkrótce nie zostanie ono zweryfikowane!
+verified_caps=Zweryfikowany
+unverified_caps=Niezweryfikowany
+resend_verification=Wyślij ponownie weryfikacje
+email_verification=Weryfikacja Email
+change_email=Zmiana Email
+change_email_address=Zmiana adresu Email
+new_email=Nowy Email
+confirm_email=potwierdź Email
+ok=ok
+security=Zabezpieczenia
+security_description=Przede wszystkim najlepszym sposobem zabezpieczenia swojego konta jest upewnienie się, że masz zweryfikowany adres e-mail. Jeśli chcesz zmienić hasło, tutaj znajdziesz odpowiednie miejsce, aby to zrobić.
+change_password=Zmiana hasła
+current_password=Aktualne Hasło
+new_password=Nowe hasło
+confirm_new_password=Potwierdź nowe hasło
+supporter_status=Supporter Status
+active_supporter_thanks_description=Jesteś osobą aktywnie wspierającą projekt! Bardzo dziękuję za wszelkie wsparcie projektu w ten sposób - tak naprawdę zostanie wspierającym to pójście o krok dalej - to osobiste wspieranie mnie, gdy podążam za moim marzeniem... to naprawdę wiele dla mnie znaczy. Dziękuję!!! <3
+active_caps=AKTYWNY
+expires_date=WYGASA {0}
+inactive_caps=NIEAKTYWNY
+my_account=Moje Konto
+purchase_history=Historia zakupów
+purchase_history_description=Mirage śledzi wszystko, co kiedykolwiek kupiłeś i możesz to wszystko sprawdzić tutaj. Ponadto, jeśli zapłaciłeś za coś, ale tego nie otrzymałeś, nie obawiaj się - tutaj możesz porównać wszystkie swoje zakupy dokonane na Twoim urządzeniu z naszymi danymi.
+redeem_purchases=Odbierz zaległe zakupy
+redeem_purchases_description=Gdy kupujesz coś w sklepie GooglePlay, transakcja jest przechowywana na Twoim urządzeniu. Oznacza to, że jeśli przypadkowo rozłączysz się w trakcie dokonywania zakupu, na Twoim urządzeniu może znajdować się zapis transakcji, ale może on nie zostać dostarczony. Jeśli tak się stanie - nie panikuj, naciśnij poniższy przycisk, aby sprawdzić, czy na Twoim urządzeniu nie ma niedostarczonych zakupów!
+check_device_purchases=Zrealizuj zakupy
+checking_purchases_dots=Sprawdzanie zakupów...
+purchases_can_only_be_redeemed_on_android=Niestety, zakupy w sklepie Play Store mogą być dokonywane tylko na urządzeniach z systemem Android :(
+history=Historia zakupów
+history_description=Tutaj możesz zobaczyć wszystko, co kupiłeś w ciągu ostatnich 30 dni - ile za to zapłaciłeś i kiedy zostało dostarczone. Proste :)
+no_unredeemed_purchases_found_on_device=Na Twoim urządzeniu nie znaleziono żadnych nieodebranych zakupów.
+no_unredeemed_purchases_found_on_device_description=
```

Comment:
There is currently an issue with the Play Store that sometimes results in it not reporting purchases correctly to Mirage. If you have made a purchase but not received it, the easiest thing to try is restarting your device (this clears your device's purchase cache) and pressing the "Redeem Purchases" button again. If this doesn't work please find your Google Play purchase receipt email and send the Order Number to us via one of the following methods:

^Tłumaczenia tego w tej linijce brakuje. Proponuję:
W Play Store obecnie niekiedy występuję błąd, który sprawia że zakupy dokonane w Mirage nie zostają odnotowane,
Jeśli dokonałeś jakiegoś zakupu i nie otrzymałeś tego co kupiłeś, najprostszym rozwiązaniem będzie zrestartowanie swojego urządzenia i kliknięcie w przycisk 'Odbierz zaległe zakupy'. Jeśli to nie poskutkuje to proszę odnaleźć e-mail od Google Play, odszukać w nim 'Numer Zamówienia' i przesłać go nam poprzez te metody: